package.json

```diff
@@ -7,23 +7,22 @@
     "url": "https://github.com/mozilla/shield-studies-addon-utils/issues"
   },
   "dependencies": {
-    "assert": "^1.4.1",
-    "fs-extra": "^4.0.0",
-    "fx-runner": "^1.0.7",
-    "geckodriver": "^1.8.0",
-    "mocha": "^3.4.2",
-    "path": "^0.12.7",
```

Comment:
`path` devDep is built-in to Node. See [this issue](https://github.com/mozilla/shield-studies-addon-template/pull/41) in the template repo.